source/compatibility.tex

```diff
@@ -2019,6 +2019,19 @@
 Translation units compiled against this version of \Cpp{} may be incompatible with
 translation units compiled against \CppXVII{}, either failing to link or having undefined behavior.
 
+\rSec2[diff.cpp17.input.output]{\ref{input.output}: input/output library}
+
+\diffrefs{forwardlist}{list}
+\change
+Character array extraction only takes array types.
+\rationale Increase safety via preventing buffer overflow at compile time.
+\effect
+Valid \CppXVII{} code may fail to compile in this International Standard:
+\begin{codeblock}
+auto p = new char[100];
+std::cin >> std::setw(20) >> p;
```

Comment:
I'd suggest to add the usual comment "ill-formed; previously well-formed" here (yes, it's not in the paper, but still).
A somewhat larger, but still editorial, change would be to show a still well-formed use, too, e.g. by having another array "q" as a local variable.

Reply:
@jensmaurer done, thanks!